### PR TITLE
[ADD] gap_analysis_project

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: gap_analysis_project/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_project}
+==============
+
+linking business requirements to project.
```
Comment (elicoidal) [inline gap_analysis_project/README.rst]: This module is part of a set ("Gap Analysis") and provides the possibility to
create tasks in a project based on the business analysis estimation content.

File: gap_analysis_project/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_project/README.rst]: No specific steps required for installation

File: gap_analysis_project/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_project/README.rst]: No specific steps required for Configuration

File: gap_analysis_project/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_project}
+==============
```
Comment (elicoidal) [inline gap_analysis_project/README.rst]: underline until the end

File: gap_analysis_project/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Known issues / Roadmap
+======================
+
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
```
Comment (elicoidal) [inline gap_analysis_project/README.rst]: modify accordingly

File: gap_analysis_project/README.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
```
Comment (elicoidal) [inline gap_analysis_project/README.rst]: modify accordingly

File: gap_analysis_project/wizard/generate_tasks.py [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+class BrGenerateTasksLine(models.TransientModel):
+    _name = 'br.generate.tasks.line'
+    _description = 'Generate Tasks Lines'
+
+    sequence = fields.Integer('Sequence')
+    name = fields.Char('Name', size=200)
+    estimated_time_total = fields.Float(
+        string='Total estimated time',
+        help='sum up all the time of the estimation of each estimation line.'
```
Comment (elicoidal) [inline gap_analysis_project/wizard/generate_tasks.py]: Sum up all the estimated times of each estimation line.

**Comment by rafaelbn** [comment]
This module depends in https://github.com/OCA/project/pull/127

**Pull request closed without merging**